tracing/plugin/otlp.go

```diff
@@ -135,6 +143,35 @@ type TraceConfig struct {
 	TraceSamplingRatio float64 `toml:"sampling_ratio,omitempty"`
 }
 
+func convertOTLPToEnv(ic *plugin.InitContext) {
+	cfg := ic.Config.(*OTLPConfig)
+
+	if cfg.Endpoint != "" && os.Getenv(otlpEndpointEnv) == "" && os.Getenv(otlpTracesEndpointEnv) == "" {
+		os.Setenv(otlpTracesEndpointEnv, cfg.Endpoint)
```

Comment:
Hold up, I may have used the wrong var here. Must realized my thinking here for the var to use was bad since the traces endpoint is technically different than the normal endpoint.

Reply:
Ok fixed this.